kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala

```diff
@@ -136,15 +136,22 @@ abstract class TFrontendService(name: String)
     }
   }
 
-  protected def getUserName(req: TOpenSessionReq): String = {
+  /**
+   * Get the real user and the proxy user.
+   * The real user is the user used for session authentication.
+   * The proxy user is the session user, if proxy user is used, it is the proxy user,
+   * otherwise, it is the real user.
+   */
+  protected def getRealUserAndProxyUser(req: TOpenSessionReq): (String, String) = {
```

Comment:
```suggestion
  /**
   * Get the real user and the session user.
   * The real user is the user used for session authentication.
   * The session user is the proxy user if proxy user is provided, otherwise is the real user.
   */
  protected def getRealUserAndSessionUser(req: TOpenSessionReq): (String, String) = {
```